UI/sign-up.html

```diff
@@ -38,7 +40,7 @@
             <input type="password" name="confirm_password" value="" placeholder="******" min="6">
           </div>
           <div class="button">
-            <button type="submit" name="signup">Sign Up</button>
+            <button type="submit" class="submit">Sign Up</button>
```

Comment:
i have not yet understood this... i think you can make one button tag and put all the attributes above together

Reply:
@kdaglas  thanks for your concern. The later is a modification of the former. I was using a submit button, but I needed to keep the flow of the page when a user clicks Sign up button that's why I changed to anchor from button.